code/game/objects/items.dm

```diff
@@ -427,6 +428,46 @@ var/global/list/items_blood_overlay_by_type = list()
 	if(I && !I.abstract)
 		I.showoff(src)
 
+/mob/living/carbon/verb/spin_in_hand()
```

Comment:
Any cooldown for this thing?

Reply:
if you do it twice in a row with an item the item flies out of your hand, if you do the grab version either it drops it or needs two willing people.